WundergroundNetLib/Model/WundergroundData.cs

```diff
@@ -4,16 +4,18 @@
 using System.Text;
 using System.Threading.Tasks;
 
+using WundergroundNetLib.Interfaces;
+
 namespace WundergroundNetLib
 {
-    public class WundergroundData
+    public class WundergroundData : IWundergroundData
     {
-        public ObservationLocation observationLocation { get; set; }
-        public CurrentConditions currentConditions { get; set; }
-        public List<Forecast> fourDayForecast = new List<Forecast>();
+        public IObservationLocation observationLocation { get; set; }
+        public ICurrentConditions currentConditions { get; set; }
+        public List<IForecast> fourDayForecast { get; set; }
```

Comment:
This is slightly different to your initial implementation, for simplicity and to ensure compilation, I've made this into a getter/setter.